addon/models/relationship.js

```diff
@@ -0,0 +1,7 @@
+import Model from 'ember-data/model';
+
+export default Model.extend({
+    target_type: DS.attr('string'),
```

Comment:
In particular, the way the ApplicationSerializer transforms these fields will probably break this (underscored -> camelized)